### DISCUSSION TITLE
Show of hands - who has blown out their global menu?

### DISCUSSION THREAD

**Discussion by eswanzey** [vanjarosoftware/Vanjaro.Platform#2299] [General]
So, you have a global site menu in the masthead, and decide to add a side menu to the page. You better remember to select "Customize This Block Only" or you'll be making changes to your global menu. I happens to me every time I haven't added a menu in a while...

Is it possible to override the default setting to be CTBO? 

![2023-01-14_08-57-38](https://user-images.githubusercontent.com/8103212/212476077-e7cd0c9b-d5a6-4067-bb24-c3ae66e02a32.jpg)

**Comment by Gaganbird** [comment]
There is no option to override default setting at this time. 

Please add your suggestion or feature request to our Community Page and be sure to vote up: https://www.vanjaro.com/resources/help-center/community

Our roadmap is community driven and we periodically review the most requested/voted items and incorporate them in future version of the product.
